migrations/Migration.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Migration
+ *
+ * @package Tutor
+ * @author Themeum <support@themeum.com>
+ * @link https://themeum.com
+ * @since 3.8.0
+ */
+
+namespace Tutor\Migrations;
+
+/**
+ * Class Migration
+ */
+class Migration {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		QuizAttemptMigrator::instance()->schedule();
```

Comment:
Once the migration is complete, you no longer need to initiate this process. Need to handle that scenario.